lib/casserver/views/login.erb

```diff
@@ -8,6 +8,17 @@
         <% if @message %>
            <p><%= escape_html @message[:message] %></p>
         <% end %>
+
+        <% if request.cookies["login_flash"] %>
+          <p><%= escape_html request.cookies["login_flash"] %></p>
+          <%
+            response.set_cookie("login_flash", {
+              :value => '',
+              :expires => Time.now,
+              :domain => '.beyondz.org'
```

Comment:
Is there an existing config value that we can read the domain from?  If not, let's make a new one.   I don't want to hardcode the domain.  Especially with the rebranding project on the horizon.